java/impl/gcp/src/main/java/co/worklytics/psoxy/SecretManagerConfigService.java

```diff
@@ -110,54 +108,39 @@ public Optional<String> getConfigPropertyAsOptional(ConfigProperty property) {
         }
     }
 
-    private String parameterName(ConfigProperty property) {
-        if (StringUtils.isBlank(this.namespace)) {
-            return property.name();
-        } else {
-            return this.namespace + property.name();
-        }
-    }
-
     @Override
     public boolean acquire(@NonNull String lockId, @NonNull java.time.Duration expires) {
         Preconditions.checkArgument(StringUtils.isNotBlank(lockId), "lockId cannot be blank");
 
-        final SecretName lockSecretName = SecretName.of(projectId, this.namespace + lockId);
-
-        long secretTtl = Math.max(expires.getSeconds(), LOCK_SECRETS_TTL.getSeconds());
+        final SecretName lockSecretName = getLockSecret(lockId);
```

Comment:
effect of this is that it's not a generic locking service; but it's actually acquiring a lock on an existing secret. Not a problem atm, but that doesn't extend to case where customer might want there secrets actually in Vault or something. (even if customer wants secrets in vault, it's perfectly reasonable to still implement locking on top of GCP Secret Manager - as lock doesn't contain sensitive info)